app/models/role.rb

```diff
@@ -149,6 +149,7 @@ def self.administrative? permission:
     permissions_with_descriptions.merge(health_permissions_with_descriptions)[permission][:administrative] rescue true # rubocop:disable Style/RescueModifier
   end
 
+  # appears to be broken, the :access property is not present in permissions_with_descriptions
   def self.permissions_for_access(access)
     permissions_with_descriptions.select { |_k, attrs| attrs[:access].include?(access) }.keys
   end
```

Comment:
Noticed this issue while investigating:
This is called by `with_viewable_permissions` but throws an exception, maybe it's rotted away and should be removed?

Reply:
Thanks, looks like there are corollary versions in the HMIS side that _are_ used.  Removing this and a couple other unused methods that called it.